engine/installation/linux/docker-ce/ubuntu.md

```diff
@@ -34,10 +34,10 @@ versions:
 - Xenial 16.04 (LTS)
 - Trusty 14.04 (LTS)
 
-Docker CE is supported on Ubuntu on `x86_64`, `armhf`, and `s390x` (IBM z
+Docker CE is supported on Ubuntu on `x86_64`, `armhf`, `ppc64le` (IBM POWER) and `s390x` (IBM z
 Systems) architectures.
 
-> **`s390x` limitations**: System Z is only supported on Ubuntu Xenial and Zesty.
+> **`ppc64le` and `s390x` limitations**: IBM Power and z Systems are only supported on Ubuntu Xenial and above.
```

Comment:
Can you rebase? I don't think this is accurate anymore.

Reply:
updated to say it's an availability issue, not a support one. https://download.docker.com/linux/ubuntu/dists/

Reply:
rebased. also I'll be out until the new year 🎉  🎄

Reply:
Enjoy the Holidays!